### PR TITLE
feat: change products category page layout

### PR DESCRIPTION
**Pull request by carylemedeho**

### CODE REVIEW

File: src/lib/frontend/components/ProductCardShadow.svelte [hunk 1]
```diff
@@ -1 +1 @@
-<div class="w-full h-[20.5rem] sm:h-[19.5rem] md:h-[16.1rem] xl:h-[14rem] bg-slate-50 animate-pulse"></div>
\ No newline at end of file
+<div class="h-[326px] p-4 rounded-10 shadow flex-col justify-start items-start gap-[25px] inline-flex w-full bg-slate-200 animate-pulse"></div>
```
Comment (hamadycisse) [inline src/lib/frontend/components/ProductCardShadow.svelte:1]: Do we need a component for a shadow? Can't this be part of the base component for product cards?

File: src/lib/frontend/components/CategoryProducts.svelte [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import type {OfferSummaryPresentation} from "$lib/frontend/presentations/OfferSummaryPresentation";
```
Comment (hamadycisse) [inline src/lib/frontend/components/CategoryProducts.svelte:2]: Remind me the difference between the CategoryProducts and the Collection

Comment (carylemedeho) [reply src/lib/frontend/components/CategoryProducts.svelte:2]: Collection contains unrelated products without any actions. CategoryProducts contains products from the same category with pagination behaviour (loadMore)

Comment (hamadycisse) [reply src/lib/frontend/components/CategoryProducts.svelte:2]: Where do we have Collections in the app right now?

**Review by hamadycisse — APPROVED** [review]

File: src/lib/frontend/components/CategoryProducts.svelte (hunk 2 above)
Comment (carylemedeho) [reply src/lib/frontend/components/CategoryProducts.svelte:2]: Home page

**Pull request merged**